src/app/pages/confirmation/confirmation.js

```diff
@@ -23,70 +24,157 @@ const models = {
+}
+
+function fetchErrataButtonsAndDetail(errataId) {
+    const slug = `errata/${errataId}`;
+    const [belowHeaderButtons, setBelowHeaderButtons] = useState([]);
+    const [detailComponent, setDetailComponent] = useState();
+
+    useEffect(async () => {
+        const detail = await fetchFromCMS(slug);
+        const detailModel = await getDetailModel(detail);
+
+        setDetailComponent(new Detail(detailModel));
+        setBelowHeaderButtons([{
+            text: `Submit ${detailModel.detail.bookTitle} errata`,
+            colorScheme: 'white-on-blue',
+            url: `/errata/form?book=${encodeURIComponent(detailModel.detail.bookTitle)}`
+        }]);
+    }, [errataId]);
+
+    return [belowHeaderButtons, detailComponent];
+}
+
+function fetchDefaultEmail(id) {
+    const [email, setEmail] = useState();
+
+    useEffect(async () => {
+        const response = await userModel.load();
 
-    onLoaded() {
-        if (this.model.popupText) {
-            this.regions.popup.attach(new Popup(this.model.popupText));
-        }
-        if (this.referringPage === 'errata') {
-            const {id} = $.parseSearchString(window.location.search);
-            const slug = `errata/${id}`;
-
-            fetchFromCMS(slug).then((detail) => {
-                getDetailModel(detail).then((detailModel) => {
-                    this.model.belowHeader.buttons = [{
-                        text: `Submit ${detailModel.detail.bookTitle} errata`,
-                        colorScheme: 'white-on-blue',
-                        url: `/errata/form?book=${encodeURIComponent(detailModel.detail.bookTitle)}`
-                    }];
-                    this.update();
-                    this.regions.detail.attach(new Detail(detailModel));
-                });
-            });
-
-            this.model.errataId = id;
-            this.userPromise.then((response) => {
-                this.model.defaultEmail = response.email;
-                this.update();
-            });
-        }
+        setEmail(response.email);
+    });
+    return email;
+}
+
+function BelowHeader({text, buttons}) {
+    return (
+        <div className="wrapper">
+            <div className="text-content">
+                <div className="below-header-text">{text}</div>
+                <div className="buttons">
+                    {
+                        buttons.map((b) =>
+                            <a className={`btn ${b.colorScheme}`} href={b.url}>
+                                {b.text}
+                            </a>
+                        )
+                    }
+                </div>
+            </div>
+        </div>
+    );
+}
+
+function ErrataStatusNotification({errataId}) {
+    if (!errataId) {
+        return null;
     }
 
+    const email = fetchDefaultEmail(errataId);
+    const notifyByEmail = email && email !== 'none@openstax.org';
+
+    return (
+        notifyByEmail ?
+            `We'll be sending submission updates to ${email}.` :
```

Comment:
do you need some kind of loading state here? email will be undefined until the user model loads

Reply:
If email is undefined, the user sees a link. If email successfully loads, that gets replaced with a message.